### ISSUE TITLE
Accessibility: error handling improvements

### ISSUE THREAD

**Issue by schnobbl** [grafana/grafana#21548]
Hi folks,

when entering some wrong data (for example a invalid format for "Evaluate every" in an alert rule) the affected field is not highlighted in GUI:

https://i.paste.pics/d18c0faed410e6dbdebc30d92763ec7c.png

A red border or sth else would help to better find the problem on the screen.

Regards,
Norbert

**Steps to reproduce**:

- Edit a Dashboard
- Create an Alert
- Enter invalid value "test" into Field "Evaluate every"
- Save the dashboard

**Actual Result**:
- error message appears for a few seconds on the top right border
- affected field is not highlighted

**Expected Result**
- error message appears permanently on the top right border
- affected field is highlighted

**Relevant WCAG Criteria:** [3.3.1 WCAG Criterion](link to https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html)

**Environment**:
- Grafana version: 6.5.2
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by MaximKipot** [comment]
We also encounter the same usability and what most important accessibility issues with the errors and alerts of the Grafana interface.
Unfortunately, accessibility and UX guidelines and best practices are not being followed.

Following the well-known error messages guidelines from Nielsen Norman Group Grafana violates at least 3 of the main basic rules for better UX. (https://www.nngroup.com/articles/errors-forms-design-guidelines/)

**Comment by ckbedwell** [comment]
![image](https://github.com/grafana/grafana/assets/6906380/7272c3d9-f373-42c9-8ea1-596fdfdc984f)

As per the details of this issue this is now resolved so I will be closing it.

It is worth noting that are some accessibility issues still related to this form but those are separate to what has been reported here.
